### ISSUE TITLE
saving a set of messages to a single file is not possible in v.14

### ISSUE THREAD

**Issue by cristina1948** [thundernest/import-export-tools-ng#477]
saving a set messages to a single file is not possible in v.14. Why? Can you restore it. I find it very useful

**Comment by cleidigh** [comment]
@cristina1948 
This is in the right click on a folder under the plaintext export.
If this is not what you mean can you post a screen shot of what menu you mean.
@cleidigh

**Comment by cristina1948** [comment]
In the previous version there was the option "export messages in the folder in a single text file" and then you could choose with or without attachment, This is what I would like to have back. Thanks, Cristina

**Comment by cristina1948** [comment]
It is the option "save the selected messages in a single file" that I would like to have back

**Comment by cleidigh** [comment]
What version?
v12.04 I just went over and there's no single file option so I am very
confused.
I will try to figure out the disconnect, but right now I am just not
understanding.
Another user commented the same and when he compared his 102 to 115 he
realized it was only in the folder menus.

It's certainly that can be added, I just can't understand the confusion.
@cleidigh

**Comment by cleidigh** [comment]
@cristina1948 
I managed to get the screenshot of the Save selected messages… menu.
This is Thunderbird v102.15.1 and ImportExportTools NG v12.0.4
The plaintext exports are individual text files. I am racking my head trying to figure out what you are seeing.

<img width="960" alt="image" src="https://github.com/thundernest/import-export-tools-ng/assets/25272315/8e133b71-093f-4e13-b007-83db2014df67">

Now this is the folder context menu where there are two single file export options (which are available in v14.0.0

<img width="960" alt="image" src="https://github.com/thundernest/import-export-tools-ng/assets/25272315/97e14799-0749-40d8-a997-084afae52516">


@sfhowes you had the same thought, but on inspection you realized v12.0.4 ONLY has the single file options in the folder context menu, yes?

@cleidigh

**Comment by sfhowes** [comment]
Yes, just checked 102 and 115.  Both versions of IET NG have export of selected messages to _individual_ text files, and both have export of a _folder_ to a single text file (or individual text files).  Just the wording and menu locations are a bit different.  Adding export of selected messages to a single text file would be nice, but as I mentioned before, it can be done by copying the selected messages to a temporary folder, then exporting the folder.

**Comment by cleidigh** [comment]
@sfhowes 
Thanks for your quick response.
I agree it should be on the possible enhancement list. I just don't understand why it seems some users thought this was an existing function.
Appreciated,
@cleidigh
